CHANGELOG.rst

```diff
@@ -1,6 +1,9 @@
 3.0.5.dev0
 ==========
 
+* Add Argparse style type validation for ``--confcutdir`` and ``--junit-xml`` (`#2089`_).
```

Comment:
Perhaps something more user oriented, for example:

```
* Now ``--confcutdir`` and ``--junit-xml`` are properly validated if they are directories and filenames, respectively (`#2089`_ and `#2078`_). Thanks to `@lwm`_ for the PR.
```

With this, you can also remove the entry "An error message is now displayed if ``--confcutdir`` is not a valid directory..." some items below this one, since it replaces it.